docs/cli/product-exporter.md

```diff
@@ -0,0 +1,115 @@
+# Product Exporter
+
+A package that helps with exporting [commercetools products](https://dev.commercetools.com/http-api-projects-products.html) from the [commercetools platform](http://dev.commercetools.com/).
+The products can be exported in JSON format, or as chunks that can be piped to a parser for more export formats.
+
+## Configuration
+
+The constructor accepts four arguments:
+- `apiConfig` (Object): `AuthMiddleware` options for authentication on the commercetools platform. (Required. See [here](https://commercetools.github.io/nodejs/sdk/api/sdkMiddlewareAuth.html#named-arguments-options))
+- `exportConfig` (Object): Internal Export configurations
+  - `batch` (Number): Amount of products to fetch for each API call
+  - `expand` (Array): An array of strings signifying reference fields to expand in the returned product
+  - `json` (Boolean): Specify if products returned should be in JSON file format. If set to false, the products will be output in chunks (Default: true)
+  - `predicate` (String): Query string specifying (where) predicate. More info on predicates [here](http://dev.commercetools.com/http-api.html#predicates) (Optional)
+  - `staged` (Boolean): Specify if prices should be fetched from all products (true) or only published products (false) (Optional. Default: false)
+  - `total` (Number): The total number of products to fetch
+- An optional logger object having four methods (`info`, `warn`, `error` and `debug`)
+- `accessToken` (String): Access token to be used to authenticate requests to API. Requires scope of [`view_products`, `view_customers`]
+
+
+## Usage
+`npm install @commercetools/product-exporter --global`
+
+### CLI
+```
+Usage: product-exporter [options]
+Export products from the commercetools platform
+
+Options:
+  --help, -h        Show help text.                                    [boolean]
+  --version, -v     Show version number                                [boolean]
+  --projectKey, -p  API project key                          [string] [required]
+  --apiUrl          The host URL of the HTTP API service
+                                     [string] [default: "https://api.sphere.io"]
```

Comment:
change this url to `https://api.commercetools.com`, the `sphere.io` one has issues.

Reply:
I get this error with `http://api.commercetools.io`:

`ERR: request to https://auth.commercetools.io/oauth/token failed, reason: getaddrinfo ENOTFOUND auth.commercetools.io auth.commercetools.io:443]`

Reply:
it's `api.commercetools.com` not `io`